python/semantic_kernel/processes/kernel_process/kernel_process_step_metadata.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+
+class KernelProcessStepMetadataAttribute:
+    """The metadata attribute for a Kernel Process Step."""
+
+    def __init__(self, version: str = "v1"):
+        """Initializes a new instance of the KernelProcessStepMetadataAttribute class."""
+        self.version = version
+
+
+def kernel_process_step_metadata(version: str = "v1"):
```

Comment:
I understand the name but I think it should be shorter, `@step_metadata` for instance...